{{ cookiecutter.project_name }}/Makefile

```diff
@@ -128,6 +144,7 @@ _install_deps:
 		--disable-pip-version-check \
 		-Ur requirements.txt -t ${SERVICE}/build/
 	@rm -f requirements.txt
+	@cp -a events ${SERVICE}/build/
```

Comment:
I've corrected this with 1c75da2

@heitorlessa Should the revised contents of line 147 moved up to the `_clone_service_to_build` make target instead of `_install_deps`?